src/app/pages/StoryPage/index.stories.jsx

```diff
@@ -10,63 +9,59 @@ import persianPageData from './fixtureData/persian';
 import { STORY_PAGE } from '#app/routes/utils/pageTypes';
 
 const withSecondaryColumnsKnob = pageData => storyFn => {
-  const options = {
-    'without Top Stories': 'topStories',
-    'without Features': 'features',
-    'without Features & Top Stories': ['features', 'topStories'],
-    default: '',
-  };
-  const selectedColumns = select(
-    'Select secondary column options',
-    options,
-    '',
-    'STY-SECONDARY-COLUMN',
-  );
+  const showTopStories = boolean('Show Top Stories', true);
+  const showFeaturedStories = boolean('Show Featured Stories', true);
 
   const secondaryColumn = {
-    ...(!selectedColumns.includes('topStories') && {
+    ...(showTopStories && {
       topStories: pageData.secondaryColumn.features,
     }),
-    ...(!selectedColumns.includes('features') && {
+    ...(showFeaturedStories && {
       features: pageData.secondaryColumn.features,
     }),
   };
 
   const storyProps = {
-    data: {
+    pageData: {
```

Comment:
Intentional change - this decorator doesn't work on Live because it's updating an unused `data` prop rather than `pageData` - https://bbc.github.io/simorgh/?path=/story/pages-story-page--mundo